RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -109,9 +114,8 @@ public bool Generate(ProcessedArgs args)
             definition.Manifest = CreatePluginManifest(package);
 
             // Create a zip containing the required analyzer files
-            definition.StaticResourceName = Path.GetFileName(packageDir) + ".zip";
-            definition.SourceZipFilePath = Path.Combine(baseDirectory, definition.StaticResourceName);
-            ZipFile.CreateFromDirectory(packageDir, definition.SourceZipFilePath, CompressionLevel.Optimal, false);
```

Comment:
The old code packed the whole NuGet package directory, which included the zipped NuGet package and the unzipped files. The new version excludes the NuGet-specific files and retains everything else.